pkg/apis/etcd/v1beta2/backup_types.go

```diff
@@ -169,3 +176,22 @@ type GCSBackupSource struct {
 	// If omitted, client will use the default application credentials.
 	GCPSecret string `json:"gcpSecret,omitempty"`
 }
+
+// OSSBackupSource provides the spec how to store backups on OSS.
+type OSSBackupSource struct {
+	// Path is the full abs path where the backup is saved.
+	// The format of the path must be: "<oss-bucket-name>/<path-to-backup-file>"
+	// e.g: "mybucket/etcd.backup"
+	Path string `json:"path"`
+
+	// The name of the secret object that stores the credential which will be used
+	// to access aliyun OSS.
```

Comment:
Describe the expected format of the secret in this comment. That it must contain the following keys/fields:
- `accessKeyID`
- `accessKeySecret `